### PR TITLE
r*: Fix `Homebrew/NoFileutilsRmrf` RuboCop offenses

### PR DESCRIPTION
**Pull request by issyl0**
- [ ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [ ] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

There's Rust in this batch. 🦀 😬

### CODE REVIEW

**Review by chenrui333 — APPROVED** [review]

**Comment by chenrui333** [comment]
> There's Rust in this batch. 🦀 😬

let's maybe wait for #178436 to be merged first.

**Comment by chenrui333** [comment]
rebased per rust PR merge

File: Formula/r/ruby.rb [hunk 1]
```diff
@@ -153,11 +153,11 @@ def post_install
     # Since Gem ships Bundle we want to provide that full/expected installation
     # but to do so we need to handle the case where someone has previously
     # installed bundle manually via `gem install`.
-    rm_f %W[
+    rm(%W[
       #{rubygems_bindir}/bundle
       #{rubygems_bindir}/bundler
-    ]
-    rm_rf Dir[HOMEBREW_PREFIX/"lib/ruby/gems/#{api_version}/gems/bundler-*"]
```
Comment (scpeters) [inline Formula/r/ruby.rb:160]: calling `brew install ruby` now fails for me in this postinstall step if these files don't exist in `rubygems_bindir`. I think the `-f` to `rm` was allowing the operation to succeed even if the files don't exist, and now they can fail

Comment (scpeters) [reply Formula/r/ruby.rb:160]: fix in https://github.com/Homebrew/homebrew-core/pull/179332

**Pull request merged**